cluster/src/test/java/org/apache/iotdb/cluster/server/member/MemberTest.java

```diff
@@ -109,6 +111,9 @@ public void setUp() throws Exception {
     IoTDBDescriptor.getInstance().getConfig().setEnableWal(false);
     RaftMember.setWaitLeaderTimeMs(10);
 
+    RaftServer.setSyncLeaderMaxWaitMs(100);
+    RaftServer.setHeartBeatIntervalMs(100);
```

Comment:
Maybe we need to save the previous settings and then reset them after the test is finished.

Reply:
Thanks 
done